packages/core/src/text/Code.tsx

```diff
@@ -0,0 +1,12 @@
+import { forwardRef } from "react";
+import { Text, TextProps } from "./Text";
+
+export const Code = forwardRef<HTMLSpanElement, Omit<TextProps<"span">, "as">>(
```

Comment:
```suggestion
export const Code = forwardRef<HTMLSpanElement, Omit<TextProps<"code">, "as">>(
```
Is it possible to use code here?